plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/TenantDataProvider.java

```diff
@@ -179,6 +118,72 @@ private String getResponseBody(final String endpoint, final String keycloakToken
     return response.asString();
   }
 
+  private class TenantDataLoader extends CacheLoader<CacheKey, UserCheTenantData> {
+
+    @Override
+    public UserCheTenantData load(CacheKey cacheKey) throws InfrastructureException {
+      if (standalone) {
+        String namespaceType = cacheKey.namespaceType;
+        String namespaceToUse;
+        switch (namespaceType) {
+          case "che":
+            namespaceToUse = cheNamespace;
+            break;
+          default:
+            namespaceToUse = "myproject";
+        }
+        return new UserCheTenantData(
+            namespaceToUse, "https://kubernetes.default.svc", "dummy.prefix.unused", false);
+      }
+
+      String responseBody;
+      try {
+        responseBody = getResponseBody(fabric8UserServiceEndpoint, cacheKey.keycloakToken);
+      } catch (ServerException
```

Comment:
Can we just use Exception instead of listing all possible exceptions here?

Reply:
We can -- I just copy-pasted :)

Fixed.